exercises/ayomide_bamidele/ex02_oo_basics/src/Circle.cpp

```diff
@@ -1,9 +1,7 @@
 #include "Circle.h"
 #include <iostream>
 
-Circle::Circle(double r) : radius(r) {
-  std::cout << "Circle Created" << std::endl;
-}
+Circle::Circle(double r) : radius(r) {}
 
 double Circle::area() const { return 3.14 * radius * radius; }
```

Comment:
There is a `M_PI` macro defined in c++ for the value of Pi that you can use. You'll need the following includes:
```cpp
#define _USE_MATH_DEFINES
#include <cmath>
```

Reply:
Yes, I thought about that to be honest. I thought it'd be best to implement the core of the problem first before worrying about `pi`

Reply:
I'll implement it

Reply:
>  I thought it'd be best to implement the core of the problem first before worrying about pi
No worries. I'm only pointing things out to make you aware 😄